src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx

```diff
@@ -320,9 +327,19 @@ export function CollapsibleNavGroupEnabled({
             hasShadow={false}
             className="eui-yScroll flex-1-container"
           >
-            {shouldShowCollapsedNavHeaderContent && collapsibleNavHeaderRender ? (
+            {shouldShowCollapsedNavHeaderContent &&
```

Comment:
Please just remove the render logic here, we can render the collapsedNavHeaderContent in the collapsible_nav_group_enabled_top.tsx.